robosherlock/src/flowcontrol/src/runAAE.cpp

```diff
@@ -135,24 +153,37 @@ int main(int argc, char *argv[])
     mongo::client::GlobalInstance instance; //this is a stupid thing we did now we suffer the consequences 
     ros::AsyncSpinner spinner(0);
 
-    engine = rs::createRSAggregateAnalysisEngine(analysis_engine_file, false);
+
+    rs::Visualizer vis(!useVisualizer, true);
     spinner.start();
+    for(auto ae_path : analysis_engine_file_paths)
+    {
+      RSAggregateAnalysisEngine* engine = rs::createRSAggregateAnalysisEngine(ae_path);
+      rsaaes.push_back(engine);
+      vis.addVisualizableGroupManager(engine->getAAEName());
+    }
     vis.start();
 
     ros::Rate rate(30.0);
     while(ros::ok())
     {
       signal(SIGINT, signalHandler);
-
-      engine->resetCas();
-      engine->processOnce();
-
-      std::vector<std::string> obj_descriptions;
-      rs::ObjectDesignatorFactory dw(engine->getCas(),rs::ObjectDesignatorFactory::Mode::CLUSTER);
-      dw.getObjectDesignators(obj_descriptions);
-      robosherlock_msgs::RSObjectDescriptions objDescr;
-      objDescr.obj_descriptions = obj_descriptions;
-      result_pub_.publish(objDescr);

```

Comment:
Where are these advertised now?

Reply:
Have reintroduced these in the last commit